features/theme.feature

```diff
@@ -249,3 +249,57 @@ Feature: Manage WordPress themes
       | name          | status   |
       | biker         | active   |
       | jolene        | parent   |
+
+  Scenario: When updating a theme --format should be the same when using --dry-run
+    Given a WP install
+
+    When I run `wp theme update twentyfifteen --version=1.0`
+    Then STDOUT should not be empty
+
+    When I run `wp theme update twentyfifteen --format=summary --dry-run`
+    Then STDOUT should contain:
+      """
+      Available theme updates:
+      Twenty Fifteen update from version 1.0 to version
+      """
+
+    When I run `wp theme update twentyfifteen --version=1.0`
+    Then STDOUT should not be empty
+
+    When I run `wp theme update twentyfifteen --format=json --dry-run`
+    Then STDOUT should contain:
```

Comment:
Let's use `Then STDOUT should be JSON containing:`, which will ensure it's valid JSON.